nnstreamer_example/example_cam/nnstreamer_example_cam.c

```diff
@@ -191,8 +191,8 @@ main (int argc, char **argv)
   /** init pipeline */
   str_pipeline =
       g_strdup_printf
-      ("v4l2src name=cam_src ! videoconvert ! "
-      "video/x-raw,width=%d,height=%d,format=RGB,framerate=30/1 ! tee name=t_raw "
+      ("v4l2src name=cam_src ! videoscale ! "
```

Comment:
Looks good. 
This PR is a next activity of https://github.com/nnsuite/nnstreamer/pull/551.